charts/templates/NOTES.txt

```diff
@@ -0,0 +1,16 @@
+
+Successfully installed OpenEBS.
+
+Check the status by running: kubectl get pods -n {{ .Release.Namespace }}
+
+The default values will install both Local and Replicated Engines. However,
```

Comment:
```suggestion
The default values install both Local and Replicated Engines. However,
```

Reply:
I guess there's no way to "template" this doc depending on what the user installed?

Reply:
Yes this doc this template friendly. What do you have in mind?

Reply:
As discussed offline, this is fine for now.